TINY-7291: Fixed autoresize incorrectly scrolling to the top of the editor when resizing (#7230)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

### Improved

> - The `editor.selection.scrollIntoView()` API now accepts no element to instead scroll the current selection into view #TINY-7291

Reviewer: ```suggestion
- The `editor.selection.scrollIntoView()` API now scrolls the current selection into view if no element is provided #TINY-7291
```
Not sure exactly how to word it, but we should try to make it clear that this is an alternative to the current behaviour, not a replacement.

Author: This is the opposite of what we want to be saying, so I don't like this suggestion sorry.

Reviewer: ```suggestion
- The `element` argument of the `editor.selection.scrollIntoView()` API is now optional, and if it is not provided the current selection will be scrolled into view instead #TINY-7291
```

Author: That seems fine if we remove the `instead` at the end?

Reviewer: updated

- Resizing table columns in some scenarios would resize the column to an incorrect position #TINY-7731
- Image resize backdrop element did not have `data-mce-bogus="all"` set #TINY-7854
- Resize handles appeared on top of dialogs and menus when using an inline editor #TINY-3263

> - Fixed the `autoresize` plugin incorrectly scrolling to the top of the editor in some cases when changing content #TINY-7291

Author: ```suggestion
- Fixed the `autoresize` plugin incorrectly scrolling to the top of the editor in some cases when changing content #TINY-7291
- Fixed the `editor.selection.scrollIntoView()`  type signature, as it incorrectly required an `Element` instead of `HTMLElement` #TINY-7291
```

@ashfordneil how does that look for a changelog entry, re the types fix?

Reviewer: LGTM
